### ISSUE TITLE
Proposal: helpful error for missing node_modules

### ISSUE THREAD

**Issue by toddw** [TrueCar/gluestick#922]
# Enhancement Proposal
This enhancement is mostly helpful to people who are newer to the javascript ecosystem. I've noticed a lot of people having problems when they clone a GlueStick app but they forget to run `yarn install` or `npm install` before running the app. The most common error message they get is an error related to failing to spawn `./node_modules/.bin/gluestick` which makes sense because it isn't there. Do you think it would be a good idea to check if the node_modules folder exists and if not we throw a more helpful error for these situations?

**Comment by zamotany** [comment]
Yeah definitely. There are also other places where more user-friendly messages are needed.
